yarn.lock

```diff
@@ -2994,7 +2994,7 @@
     "@types/minimatch" "*"
     "@types/node" "*"
 
-"@types/google-protobuf@^3.10.0", "@types/google-protobuf@^3.7.2":
```

Comment:
This change was there because dependabot doesn't have access to e/web. But every dev who has e cloned will now update yarn.lock when running `make teleport`.

Reply:
Can we track this dependency in OSS somehow, like we do in `e_imports.go` ?

Reply:
We're [discussing it on Slack](https://gravitational.slack.com/archives/C02DQ1C2BMW/p1675796791299939) at the moment. This particular dep difference could be solved by getting both e and OSS to the same version.

Reply:
That seems to be best, yes. I'll let y'all figure it out.